scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -0,0 +1,436 @@
+        return client.getTaskResultsByTag(jobId, taskTag);
+    }
+
+    public List<TaskResult> getTaskResultsByTag(String jobId, String taskTag) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.getTaskResultsByTag(jobId, taskTag);
+    }
+
+    public TaskResult getTaskResult(JobId jobId, String taskName) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.getTaskResult(jobId, taskName);
+    }
+
+    public TaskResult getTaskResultFromIncarnation(String jobId, String taskName, int inc) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.getTaskResultFromIncarnation(jobId, taskName, inc);
+    }
+
+    public boolean removeJob(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.removeJob(jobId);
+    }
+
+    public void listenJobLogs(String jobId, AppenderProvider appenderProvider) throws NotConnectedException, UnknownJobException, PermissionException {
+        client.listenJobLogs(jobId, appenderProvider);
+    }
+
+    public boolean killJob(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.killJob(jobId);
+    }
+
+    public boolean killTask(String jobId, String taskName) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.killTask(jobId, taskName);
+    }
+
+    public boolean restartTask(String jobId, String taskName, int restartDelay) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.restartTask(jobId, taskName, restartDelay);
+    }
+
+    public boolean restartInErrorTask(String jobId, String taskName) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.restartInErrorTask(jobId, taskName);
+    }
+
+    public boolean preemptTask(String jobId, String taskName, int restartDelay) throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        return client.preemptTask(jobId, taskName, restartDelay);
+    }
+
+    public boolean pauseJob(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.pauseJob(jobId);
+    }
+
+    public boolean restartAllInErrorTasks(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.restartAllInErrorTasks(jobId);
+    }
+
+    public boolean resumeJob(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.resumeJob(jobId);
+    }
+
+    public void changeJobPriority(String jobId, JobPriority priority) throws NotConnectedException, UnknownJobException, PermissionException, JobAlreadyFinishedException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)